src/main/kotlin/blackjack/domain/BlackJackPlayResult.kt

```diff
@@ -7,13 +7,16 @@ enum class BlackJackPlayResult(val resultString: String) {
     ;
 
     companion object {
+        private const val BLACKJACK_NUMBER = 21
+
         fun getResult(
-            myScore: Int,
-            opponentScore: Int,
+            normalPlayerScore: Int,
+            DealerScore: Int,
```

Comment:
```suggestion
            dealerScore: Int,
```
으로 수정해주면 좋을 것 같네요! :)